src/utils/MultipartFormDataParser.ts

```diff
@@ -3,10 +3,20 @@ import util from 'util';
 
 import { getEnvOrThrow } from 'app/utils/EnvironmentUtils';
 
+const supportedFileTypes: string[] = getEnvOrThrow('SUPPORTED_MIME_TYPES').split(',');
```

Comment:
Please double check that application does not start without `SUPPORTED_MIME_TYPES` env var.